benchmarks/performance-poetry/complex-benchmark/src/main/java/com/squareup/benchmarks/performance/poetry/complex/benchmark/ComplexPoetryResults.txt

```diff
@@ -0,0 +1,106 @@
+
+April 13
+Nexus 6
+
+Commit SHA - 7fe2fe3cbe2ef6908ce60a7876122089aaae8503
```

Comment:
Really want to commit this?

Reply:
where else do you want to store the results?